lib/helpers/jaro.dart

```diff
@@ -8,7 +8,10 @@ import 'package:multiple_search_selection/helpers/extensions.dart';
 /// its inverse, called the Jaro similarity, is a measure of two strings' similarity:
 ///
 /// the higher the value, the more similar the strings are.
-double getJaro(String s, String t) {
+double getJaro(String a, String b, { bool caseSensitive = false }) {
+  final s = caseSensitive ? a : a.toLowerCase();
+  final t = caseSensitive ? b : b.toLowerCase();
```

Comment:
this changes the current default behavior of this function, which is not something i generally like to do, but i thought keeping it consistent was more important